_about/ROADMAP.md

```diff
@@ -0,0 +1,64 @@
+# Q3 2020 Roadmap
+
+Each quarter the team will highlight areas of focus for our work and upcoming research.
+ 
+We select items to include in the roadmap from community issues and internal priorities. When community pull requests exist for an item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp may complete any additional work needed. 
+
+Each release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. 
+
+To make contribution easier, we’ll be using the [`Help Wanted`](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) tag to point to issues we’d like to include in this quarter’s series of releases. 
```

Comment:
Can you also link out to the contribution guide for people who see this document and want to pick up something in help-wanted?

Reply:
Great idea